.dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
+vendor
```

Comment:
don't we need to ignore build directory?

Reply:
This is the `.dockerignore` file. We explicitly need the build directory for doing the Docker build. :)

Reply:
@jarrpa why we need build folder(it only contains the glusterfs-csi-driver binary) for docker build?